server/routes/config/taskListErrors.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  NO_OFFENDER_NUMBER: 'Offender number required but it is not entered in Delius',
+  NO_COM_ASSIGNED: 'A Community Offender Manager has not been assigned. Please contact the Delius team',
+  LDU_INACTIVE: 'The Local Delivery Unit is in a geographical area that is currently inactive. Please refer via Nomis',
+  COM_NOT_ALLOCATED:
+    'The Community Offender Manager (COM) assigned is a pseudo. Please arrange for an actual COM to be assigned',
+}
```

Comment:
nice 👍 

Would be good for Louise to review or provide alternatives.. but obvs doesn't need to hold up this PR

Reply:
ok